pkg/app/piped/executor/analysis/analysis.go

```diff
@@ -117,7 +117,7 @@ func (e *Executor) Execute(sig executor.StopSignal) model.StageStatus {
 				}
 				status = model.StageStatus_STAGE_SKIPPED
 				// Stop the context to cancel all running analysises.
-				ctxWithTimeout.Done()
+				cancel()
```

Comment:
as convention, should rename this stop function as `stop()`

Reply:
Is this a convention for PipeCD team?
I guess `cancel()` is often used in this case.

Reply:
Ah, sorry, just my tatse. For `WithCancel` will use `cancel()` and for `WithTimeout` will use `stop()`. But not a big deal, if other use the same 👀